src/util/http-util.ts

```diff
@@ -10,14 +11,24 @@ axios.interceptors.request.use(
     if (token) {
       config.headers.Authorization = `Bearer ${token}`
     }
+    // we would want loading status for post/put/patch for now
```

Comment:
I hope we need to add one more config value to ignore loader incase needed. may be disableLoader default as false, and when we pass true we can skip loading part.